backends/core/models.py

```diff
@@ -114,14 +138,23 @@ def assign_schedules(self, arg_schedules):
         ids_for_removing.append(schedule.id)
     Schedule.destroy(*ids_for_removing)
 
+  def get_ready(self):
+    cache_values = {
+      "failed_jobs": 0,
+      "remaining_jobs": len(self.jobs.all()),
+      "list_of_tasks_enqueued": []
+    }
+    self._add_multi_to_memcache(cache_values)
```

Comment:
We can remove the mention of memcache here, since it's a choice that could be changed in the future.
Maybe something like `_add_or_update_multi_to_cache` would be better suited, what do you think?

Reply:
done